.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,494 @@
+settings_guide_synonyms_1: |-
+  let mut synonyms = std::collections::HashMap::new();
+  synonyms.insert("sweater".to_string(), vec![String::from("jumper")]);
+  synonyms.insert("jumper".to_string(), vec![String::from("sweater")]);
+
+  let top_index: Index = client.get_index("top").await.unwrap();
+  let progress: Progress = top_index.set_synonyms(&synonyms).await.unwrap();
+settings_guide_stop_words_1: |-
+  let progress: Progress = movie_index.set_stop_words(&["the", "a", "an"]).await.unwrap();
+settings_guide_attributes_for_faceting_1: |-
+  let progress: Progress = movie_index.set_attributes_for_faceting(&["director", "genre"]).await.unwrap();
+settings_guide_ranking_rules_1: |-
+  let ranking_rules = &[
+    "typo",
+    "words",
+    "proximity",
+    "attribute",
+    "wordsPosition",
+    "exactness",
+    "asc(release_date)",
+    "desc(rank)",
+  ];
+
+  let progress: Progress = movie_index.set_ranking_rules(ranking_rules).await.unwrap();
+settings_guide_distinct_1: |-
+  let jacket_index: Index = client.get_index("jackets").await.unwrap();
+  let progress: Progress = jacket_index.set_distinct_attribute("product_id").await.unwrap();
+settings_guide_searchable_1: |-
+  let searchable_attributes = &[
+    "uid",
+    "movie_id",
+    "title",
+    "description",
+    "poster",
+    "release_date",
+    "rank"
+  ];
+
+  let progress: Progress = movie_index.set_searchable_attributes(searchable_attributes).await.unwrap();
+settings_guide_displayed_1: |-
+  let displayed_attributes = &[
+    "title",
+    "description",
+    "poster",
+    "release_date",
+    "rank"
+  ];
+
+  let progress: Progress = movie_index.set_displayed_attributes(displayed_attributes).await.unwrap();
+settings_guide_accept_new_fields_1: |-
+  let progress: Progress = movie_index.set_accept_new_fields(false).await.unwrap();
+documents_guide_add_movie_1: |-
+  // Define the type of our documents
+  #[derive(Serialize, Deserialize, Debug)]
+  struct IncompleteMovie {
+    id: String,
+    title: String
+  }
+  impl Document for IncompleteMovie {
+    type UIDType = String;
+    fn get_uid(&self) -> &Self::UIDType { &self.id }
+  }
+
+  // Add a document to our index
+  let progress: Progress = movie_index.add_documents(&[
+    IncompleteMovie {
+      id: "123sq178".to_string(),
+      title: "Amelie Poulain".to_string(),
+    }
+  ], None).await.unwrap();
+search_guide_1: |-
+  // Build the query
+  let query: Query = Query::new("shifu")
+    .with_limit(5)
+    .with_offset(10);
+  
+  // Apply the query and get results
+  let results: SearchResults<Movie> = movie_index.search(&query).await.unwrap();
+search_guide_2: |-
+  // Build the query
+  let query: Query = Query::new("Avengers")
+    .with_filters("release_date > 795484800");
+  
+  // Apply the query and get results
+  let results: SearchResults<Movie> = movie_index.search(&query).await.unwrap();
+getting_started_create_index_md: |-
+  ```toml
+  [dependencies]
+  meilisearch-sdk = "0.2"
```

Comment:
```suggestion
  meilisearch-sdk = "0.3.0"
```

Reply:
Plus, #44 makes us sure that the docs will always up-to-date with the last version

Reply:
Yes, but the patch version is not needed. You can write directly "0.3".